portal-web/test/functional/com/liferay/portalweb/tests/enduser/abtest/ABTesting.testcase

```diff
@@ -1010,48 +1010,160 @@ definition {
-	@ignore = "true"
 	@priority = "3"
 	test DeleteABTestCancel {
+		task ("Create AB test and a variant") {
+			ABTest.createABTest(
+				goal = "Bounce Rate",
+				testTitle = "AB test title");
+
+			ABTest.createVariant(variantName = "Variant name");
+		}
+
+		task ("try to Delete AB Test and click cancel button") {
+			Click(locator1 = "ABTest#KEBAB_AB_TEST");
+
+			ClickNoError(
+				locator1 = "ABTest#KEBAB_OPTIONS_AB_TEST",
+				value = "Delete");
+		}
 
-		// TODO LPS-144027 DeleteABTestCancel pending implementation
+		Confirm.dismissAlert();
 
+		task ("Assert that the ABTest draft keeps appearing") {
+			AssertTextEquals(
+				key_title = "AB test title",
+				locator1 = "ABTest#AB_TEST_TITLE_DXP",
+				value1 = "AB test title");
+		}
 	}
 
 	@description = "LPS-99421: The test deletion in AC is reflected on DXP"
-	@ignore = "true"
 	@priority = "4"
 	test DeleteABTestInACViewInDXP {
+		property portal.acceptance = "true";
+
+		task ("Create AB test and create variant.") {
+			ABTest.createABTest(
+				goal = "Bounce Rate",
+				testTitle = "AB test title");
+
+			ABTest.createVariant(variantName = "Variant name");
+		}
+
+		task ("Check the info is correct in Analytics Cloud") {
+			ABTest.openABTestInAC();
+
+			ABTest.statusABTestInAC(status = "Test Is in Draft Mode");
+		}
+
+		task ("Delete AB Test at AC") {
+			Click(locator1 = "ABTest#DROPDOWN_TOGGLE_AB_TEST_AC");
+
+			ClickNoError(
+				locator1 = "ABTest#KEBAB_OPTIONS_AB_TEST",
+				value = "Delete Test");
 
-		// TODO LPS-144027 DeleteABTestInACViewInDXP pending implementation
+			Button.clickDelete();
+		}
 
+		task ("Go back to DXP") {
+			Navigator.openURL();
+
+			Navigator.openWithAppendToBaseURL(urlAppend = "web/test-site-name/content-page");
+		}
+
+		task ("Assert that does not exist ab test") {
+			ABTest.openABTestPanel();
+
+			AssertElementPresent(locator1 = "ABTest#CREATE_TEST_BUTTON");
+		}
 	}
 
 	@description = "LPS-99421: The test deletion in DXP is reflected on AC"
-	@ignore = "true"
 	@priority = "4"
 	test DeleteABTestInDXPViewInAC {
+		property portal.acceptance = "true";
+
+		task ("Create AB test and create variant.") {
+			ABTest.createABTest(
+				goal = "Bounce Rate",
+				testTitle = "AB test title");
+
+			ABTest.createVariant(variantName = "Variant name");
+		}
+
+		task ("Check the info is correct in Analytics Cloud") {
+			ABTest.openABTestInAC();
+
+			ABTest.statusABTestInAC(status = "Test Is in Draft Mode");
+		}
+
+		task ("Go back to DXP") {
+			Navigator.openURL();
+
+			Navigator.openWithAppendToBaseURL(urlAppend = "web/test-site-name/content-page");
+		}
+
+		task ("Delete the AB test") {
+			ABTest.openABTestPanel();
+
+			ABTest.editOrDeleteABTest(value = "Delete");
+		}
+
+		task ("Go to AC page and assert the ERROR page is open") {
+			Navigator.selectSpecificWindow(windowTitle = "Tests - Analytics Cloud");
 
-		// TODO LPS-144027 DeleteABTestInDXPViewInAC pending implementation
+			Refresh();
```

Comment:
Unneeded

Reply:
When I go back to the AC page to confirm the error page, I need to refresh the page so that it appears. Until I do this, the page with the previous information continues. How can I replace Refresh()?

Reply:
You're right